test/Engine/ProtoTest/LiveRunnerTests/MicroFeatureTests.cs

```diff
@@ -852,6 +852,93 @@ public void GraphILTest_DeletedNode01()
 
         }
 
+        [Test]
+        public void GraphILTest_DeletedBinaryExpresionDoesNotEffectReferences()
```

Comment:
Effectively this test is simulating the same thing that would happen in the Dynamo UI when running the graph, deleting the node, and rerunning it or is it testing something additionally?

Reply:
simulating the same thing as in the UI - I verified it failed before this change.